docs/usage.md

```diff
@@ -6,6 +6,10 @@
 
 ## Introduction
 
+<p align="center">
+    <img src="images/phyloplace_workflow.png" alt="nf-core/phyloplace workflow overview" width="60%">
```

Comment:
Please add a light gray (not white) rectangular background to thie diagram, otherwise the black dots are hard to see in the dark mode

Reply:
Thanks, done.

Reply:
It has the background, I believe. Would you prefer a link to the svg rather than the png and then remove the png from the repo?

Reply:
The svg doesn't render well in my browser. There's no background and the file icons get corrupted.

Reply:
:+1: Thanks!